app/controllers/api/messaging.php

```diff
@@ -3270,13 +3270,14 @@
     ->param('cc', null, new ArrayList(new UID()), 'Array of target IDs to be added as CC.', true)
     ->param('bcc', null, new ArrayList(new UID()), 'Array of target IDs to be added as BCC.', true)
     ->param('scheduledAt', null, new DatetimeValidator(requireDateInFuture: true), 'Scheduled delivery time for message in [ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html) format. DateTime value must be in future.', true)
+    ->param('attachments', null, new ArrayList(new CompoundUID()), 'Array of compound ID strings of bucket IDs and file IDs to be attached to the email. They should be formatted as <BUCKET_ID>:<FILE_ID>.', true)
```

Comment:
It could be confusing params are out of order with the create methods, not just attachments. Let's add a task to make all the param orders consistent in the next minor verison

Reply:
Created https://github.com/appwrite/appwrite/issues/7919